src/NuGet/EntityFramework.CodeTemplates.VisualBasic/EntityFramework.CodeTemplates.VisualBasic.nuspec

```diff
@@ -4,12 +4,12 @@
     <id>EntityFramework.CodeTemplates.VisualBasic</id>
     <version>$NuGetPackageVersion$</version>
     <title>EntityFramework.CodeTemplates.VisualBasic</title>
-    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=386613</iconUrl>
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <tags>Microsoft EF EntityFramework</tags>
     <projectUrl>http://go.microsoft.com/fwlink/?LinkID=$ProjectUrlFwLinkID$</projectUrl>
     <licenseUrl>http://go.microsoft.com/fwlink/?LinkID=$LicenseUrlFwLinkID$</licenseUrl>
```

Comment:
Again, Remove everything set in https://github.com/dotnet/arcade/blob/dd885745172424fc8df97295514f693efaec8d24/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.targets#L102-L118 and add `$CommonMetadataElements$`. Compare the final result 😺